packages/stockflux-watchlist/src/App.js

```diff
@@ -1,15 +1,9 @@
 import React from 'react';
 import Watchlist from './components/watchlist/Watchlist';
-import Components from 'stockflux-components';
 import './App.css';
 
 const App = () => {
-  return (
-    <>
-      <Components.Titlebar title="Watchlist" />
-      <Watchlist />
-    </>
-  );
+  return <Watchlist />;
```

Comment:
Maybe we should move logic from Watchlist.js into this file? For it to be similar to other apps.

Reply:
Changed the direction of the refactor as discussed.